storage/client/queries/queries.go

```diff
@@ -170,6 +170,20 @@ const (
 		LIMIT $2::bigint
 		OFFSET $3::bigint`
 
+	DelegationsTo = `
+		-- delegatee_info is used to calculate the escrow amount of the delegators in base units
+		WITH delegatee_info AS (
+			SELECT escrow_balance_active, escrow_total_shares_active
+			FROM chain.accounts
+			WHERE address = $1::text
+		)
+	        SELECT delegator, shares, delegatee_info.escrow_balance_active, delegatee_info.escrow_total_shares_active
+			FROM chain.delegations, delegatee_info
+			WHERE delegatee = $1::text
+		ORDER BY delegator, shares
+		LIMIT $2::bigint
```

Comment:
Note: This does require a sequential scan on the table. We currently have 60k delegations and the query takes ~250ms for the top delegatee (with ~60k delegators). I think we're far enough from a catastrophically slow query (nothing quadratic going on) that we can leave as-is for now.

Reply:
If this ever gets problematic, one simple fix would be to include `shrares` in the "delegator"/"delegatee" indexes, since at least for now we always sort by shares first.

Let me know if I should add it in a follow-up PR.